src/views/CardStack.js

```diff
@@ -199,30 +195,28 @@ class CardStack extends Component {
     animatedSubscribeValue(props.position);
   }
 
-  _reset(resetToIndex: number, velocity: number): void {
-    Animated.spring(this.props.position, {
+  _reset(resetToIndex: number, duration: number): void {
+    Animated.timing(this.props.position, {
       toValue: resetToIndex,
-      duration: ANIMATION_DURATION,
+      duration,
```

Comment:
The core reason for this PR is that `Animated.spring` with `bounciness = 0`  will introduce a bouncy gap _after_(visually) transition animation